desk/app/activity.hoon

```diff
@@ -476,6 +476,7 @@
   ?:  (gth time real-start)  [~ | acc]
   :-  ~   :-  |
   =/  =source:a  (source:evt -.event)
+  ?.  (~(has by indices) source)  [~ | acc]
```

Comment:
This `mint-nice`s and shouldn't be passing CI. We need to bring backend build checks back.

```suggestion
  ?:  (gth time real-start)  [~ | acc]
  :-  ~   :-  |
  =/  =source:a  (source:evt -.event)
  ?.  (~(has by indices) source)  acc
```

Or, more invasively:

```suggestion
  :+  ~  |
  ?:  (gth time real-start)  acc
  =/  =source:a  (source:evt -.event)
  ?.  (~(has by indices) source)  acc
```